distribution/docs/src/main/resources/_contents/_developing/developing-catalog-api-contents.adoc

```diff
@@ -0,0 +1,20 @@
+
+=== Catalog Development Fundamentals
+
+This section introduces the fundamentals of working with the Catalog API the OGC Filter for Queries.
```

Comment:
yes to adding the `and` & it is a capital O. (I think github's `monospace` font puts a slash through the `0`.